src/coreclr/src/inc/corinfoinstructionset.h

```diff
@@ -264,6 +276,8 @@ inline CORINFO_InstructionSetFlags EnsureInstructionSetFlagsAreValid(CORINFO_Ins
             resultflags.RemoveInstructionSet(InstructionSet_POPCNT);
 #endif // TARGET_AMD64
 #ifdef TARGET_X86
+        if (resultflags.HasInstructionSet(InstructionSet_SSE) && !resultflags.HasInstructionSet(InstructionSet_X86Base))
```

Comment:
Were these hand added or did you update the templates and rerun the generator?

Reply:
I ran the generator.  Sorry, I should have separated the generated files into their own commit to make review easier.